raiden/tests/fixtures/__init__.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+import random
+
+
+@pytest.fixture
+def get_random_bytes():
```

Comment:
this is really silly, there is no reason to use a fixture for this, just add the standalone function in the tests utils